src/components/Event.js

```diff
@@ -16,29 +17,30 @@ class Event extends Component {
       url: props.url,
```

Comment:
What's the point of copying props to state?

Reply:
fixed